### PR TITLE
Chloe/create search page

### LINKED ISSUE #48: Create Listing Summary Card Component
## Feature Description

- Params:
  - Listing Summary Object(OpenAPI spec)
  - *Note: Make it so that the `Not Interested`/`Edit Listing` button can be toggled in case of changes. 

relates: #xxx
blocked by: #xxx

## Problem You're Trying to Solve

Develop Listing Card component. Listing cards will appear on homepage, profile page (under `Your Listings`), and as a carousel for `Recommendations`.

## Related External Resources

Image

### LINKED ISSUE #54: Create Filters Component
## Feature Description

This is the component that represents the filter options underneath the search bar
- Params:
  -  *Note: Make this so that default values can be changed to other values

relates: #77 
blocked by: #45 #42 

## Problem You're Trying to Solve

Develop a component to allow user to a different filters to their searches 

## Related External Resources

Image

### LINKED ISSUE #55: Create Navbar Component
## Feature Description

This is the navbar component, this includes the icon, search bar input, filter button, search button, messaging button, and account button.
- Params: 
  - NONE   

relates: #71 #72 #75 #77
blocked by: #45 #42

## Problem You're Trying to Solve

This component will allow the user to navigate the application to the different available pages and make searches 

## Related External Resources

Image

### LINKED ISSUE #56: Create Pagination Component
## Feature Description

This is the Pagination component for changing what listings are currently being viewed in a query result.
- Params:
  - Integer: Page Number
  - Integer: Item Limit 
 

relates: #77 

## Problem You're Trying to Solve

We need a mechanism to browse through results of listings summaries and other large data.

## Related External Resources

Image

### LINKED ISSUE #71: 𝗣𝗮𝗴𝗲: Create Home/Main/Dashboard Page
## Feature Description

**View Listings: Users should be able to input their current location and view available items with pagination within a 5KM radius. They should be able to filter the items based on price and status (SOLD, AVAILABLE) and sort them by price, listed time and distance.**
This page should be created to facilitate users who have just been authenicated(from #69), it will contain a page of recommended items based on results from recommendation engine built by Algorithms. This will require components of: 
- #55 and
- #48 

_Note previously viewed listings will likely not be present on the main page as seen in the mockup below_

relates: #45, #42, #55, #48 
blocked by: #45, #42, #55, #48 

## Problem You're Trying to Solve

[See "See Personal Recommendations"](https://docs.google.com/document/d/1DNqexMBf4MpYPCxWsT3plUDAjDSbdJwUzw-UyZX-vDs/edit#heading=h.yl6jkxal18dy) and ["Indicate Disinterest"](https://docs.google.com/document/d/1DNqexMBf4MpYPCxWsT3plUDAjDSbdJwUzw-UyZX-vDs/edit#heading=h.h94p36c29gn1) from the SRS for User Stories

## Related External Resources

See [Get Recommendations](http://market.lebl.ca/openapi/#/Search%20and%20Recommendations/recommend) and [Listing Summary Object](http://market.lebl.ca/openapi/#/:~:text=Image-,ListingSummary,-LoginRequest) from the OpenAPI spec 

Image

### LINKED ISSUE #77: 𝗣𝗮𝗴𝗲: Create Search Page
## Feature Description

**View Listings: Users should be able to input their current location and view available items with pagination within a 5KM radius. They should be able to filter the items based on price and status (SOLD, AVAILABLE) and sort them by price, listed time and distance.**
This 

relates: #xxx
blocked by: #xxx

### PR DESCRIPTION
**Pull request by ChloeZacharias**
# Description

Created the home page and search page
image

image

image

Closes #71
Closes #77 
Closes #54
Closes #48 
Closes #55  
Closes #56

## Checklist
- [ ] The code includes tests if relevant
- [ ] I have *actually* self-reviewed my changes and done QA

### CODE REVIEW

File: apps/data/database/connect.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+PGPASSWORD=johuigytfrdcv psql -h localhost -p 8310 -d martletplace -U martletplace
```
Comment (sejal-agarwal) [inline apps/data/database/connect.sh:1]: Typically database passwords are stored as environment variables in a .env file which aren't pushed to the repo

File: apps/frontend/src/components/searchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        />
+      </Grid>
+      <Grid item>
+        <Button
+          type="submit"
+          variant="contained"
+          sx={{
+            ...classes.button,
+            ...classes.whiteButton,
+            backgroundColor: showFilters ? "#808080" : "white",
```
Comment (sejal-agarwal) [inline apps/frontend/src/components/searchBar.tsx:225]: Nit: Can we add #808080 as a color to the colors.tsx file? I think there is a `martletplaceWhite` in that file you can also use?

This is just a nice to have so others can use the same colors.

File: apps/data/database/initdb.sql [hunk 3]
```diff
@@ -0,0 +1,93 @@
+CREATE FUNCTION trigger_update_modified()
```
Comment (sejal-agarwal) [inline apps/data/database/initdb.sql:1]: I may be misunderstanding something, but I'm not sure why there are sql commands here. Is this for testing?

**Comment by Scott-Kenning** [comment]
Waiting to review this until the files that aren't needed in this PR are removed

**Pull request closed without merging**